CRM/Upgrade/Incremental/php/FiveFiftySeven.php

```diff
@@ -29,6 +29,21 @@ class CRM_Upgrade_Incremental_php_FiveFiftySeven extends CRM_Upgrade_Incremental
    */
   public function upgrade_5_57_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Add in Client Removed Activity Type', 'addCaseClientRemovedActivity');
+  }
+
+  public static function addCaseClientRemovedActivity() {
+    CRM_Core_BAO_OptionValue::ensureOptionValueExists([
+      'option_group_id' => 'activity_type',
+      'name' => 'Case Client Removed',
+      'label' => ts('Case Client was removed from Case'),
+      'description' => ts('Case client was removed from a case'),
+      'is_active' => TRUE,
+      'filter' => 1,
+      'component_id' => CRM_Core_Component::getComponentID('CiviCase'),
```

Comment:
Administrivia:
* This returns null if civicase is not enabled, so it would become a contact component.
* Also none of the other activity types have filter=1.